### PR TITLE
Community site

### PR DESCRIPTION
**Pull request by edewit**
introduced a new `/index.html` that can server as the start of a community page

screenshot from 2018-01-08 11-52-47
new index 
screenshot from 2018-01-08 11-53-20
getting started page

### CODE REVIEW

**Comment by gastaldi** [comment]
There is an error while generating the community page:
```
Processing community-page
[91mmaster.xml:24: parser error : Opening and ending tag mismatch: img line 19 and section
  </section>
            ^
master.xml:25: parser error : Opening and ending tag mismatch: section line 18 and article
master.xml:25: parser error : Premature end of data in tag article line 3
[0mFailed to validate community-page.
```

**Comment by tradej** [comment]
Please do not merge yet; changing the output directory of the book build scripts might break CI or other scripts, please afford me some time to investigate.

**Comment by edewit** [comment]
it doesn't change the output, just creates a sub directory

**Comment by tradej** [comment]
I know, I just want to check we don't need to change any paths in our scripts.

**Comment by tradej** [comment]
I believe the scripts will indeed work fine.

Having said that, both the `cico_build_deploy.sh` and `cico_build_test.sh` scripts run a command that is no longer necessary: `mkdir ${TARGET_DIR}/images`. These invocations should be removed if this PR is accepted in its current state, so please:

a) Remove the invocations from said scripts in this PR.
b) The person who merges this PR: Please file a follow-up issue.

Thank you.

**Comment by rhoads-zach** [comment]
@edewit @tradej 

I made some fixes here:
https://github.com/edewit/appdev-documentation/pull/4

File: docs/community-page/master.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+}
+-->
+</style>
+<section class="jumbotron">
+    <img src="/docs/images/launch_logo.svg" alt="Fabric8 Launcher Logo">
+    <div id="repoInfo">
+      <h1 class="display-4">Fabric8 Laucher</h1>
```
Comment (bartoszmajsak) [inline docs/community-page/master.adoc]: Typo. Should be `Launcher`

**Comment by rhoads-zach** [comment]
LGTM. 

any other feedback @tradej ?

**Comment by rhoads-zach** [comment]
merging.

**Comment by rhoads-zach** [comment]
merged: 1727e3cc6dbdade512f43c8279c396bf0f764d90

**Pull request closed without merging**